clients/StateChannelWallet.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers } from 'ethers';
+import { UserOperation } from './types';
+
+export class StateChannelWallet {
+  private intermediaryUrl: string;
```

Comment:
nit but I don't think we need to bother with any urls at this point.  Once we start thinking about messaging we can add addresses/urls as needed.

Reply:
That url is not currently use. I will remove it